docs/modules/llms/integrations/petals_example.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Petals LLM Example\n",
+    "This notebook goes over how to use Langchain with [Petals](https://github.com/bigscience-workshop/petals)."
```

Comment:
What do you think about including a note that it'll download ~8GB for this? I think this would require the user to have a GPU as well but not certain.

This info is from the Colab getting started tutorial linked on their github page:
https://colab.research.google.com/drive/1Ervk6HPNS6AYVr3xVdQnY5a-TjjmLCdQ?usp=sharing
![image](https://user-images.githubusercontent.com/29989939/218440626-bb0caa2c-8f26-49c0-b862-3a78ec4b7f6d.png)

Reply:
Petals does not require a GPU for inference. I can look into adding functionality for automatically checking if a GPU is available for inference.